src/shogun/clustering/KMeansBase.cpp

```diff
@@ -314,6 +415,7 @@ void KMeansBase::init()
 	    &fixed_centers, "fixed_centers", "Whether to use fixed centers",
 	    ParameterProperties::HYPER | ParameterProperties::SETTING);
 	SG_ADD(&R, "radiuses", "Cluster radiuses", ParameterProperties::MODEL);
+	SG_ADD(&stds, "stds", "Cluster standard deviations", ParameterProperties::MODEL);
```

Comment:
it is not really a model parameter right? Rather something that can be computed after having trained the algorithm

Reply:
I made it similarly to R, will change it now.